src/components/manager/projects/ProjectJsonSchema.js

```diff
@@ -0,0 +1,66 @@
+import React, { useCallback } from "react";
+import { projectJsonSchemaTypesShape } from "../../../propTypes";
+import { Button, Card, Descriptions, Modal, Typography } from "antd";
+import ReactJson from "react-json-view";
+import { useDispatch } from "react-redux";
+import { deleteProjectJsonSchema } from "../../../modules/metadata/actions";
+
+const ProjectJsonSchema = ({ projectSchema }) => {
+
+    const dispatch = useDispatch();
+
+    const handleDelete = useCallback(() => {
+        const deleteModal = Modal.confirm({
+            title: `Are you sure you want to delete the "${projectSchema.schema_type}" project JSON schema?`,
+            content: <>
+                <Typography.Paragraph>
+                    Doing so will mean that extra_properties data validation will not be
```

Comment:
Wrap `extra_properties` in a `<Typography.Text code={true}>...</Typography.Text>`

Reply:
Because Tooltip has a dark backgound, `<Typography.Text code={true}>...</Typography.Text>` is not readable.
I went with a span that has the same styling, but with white text and border. See `ExtraPropertiesCode` component in `ProjectJsonSchema.js`

Reply:
In this case, the `tooltip` prop of `ExtraPropertiesCode` is false so a `<Typography.Text code>...</Typography.Text>` is used.